packages/flutter/lib/src/material/material_button.dart

```diff
@@ -462,4 +453,5 @@ class MaterialButton extends StatelessWidget {
 ///
 /// This mixin only exists to give the "label and icon" button widgets a distinct
 /// type for the sake of [ButtonTheme].
+// TODO(Piinks): Should this be deprecated?
```

Comment:
This is only used by the buttons that are being removed, should it be deprecated @HansMuller? I checked customer references as well, no one is using this.

Reply:
Yes